event-hubs.api.md

```diff
@@ -0,0 +1,274 @@
+## API Report File for "@azure/event-hubs"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { AbortSignalLike } from '@azure/abort-controller';
+import { MessagingError } from '@azure/core-amqp';
+import { RetryOptions } from '@azure/core-amqp';
+import { Span } from '@azure/core-tracing';
+import { SpanContext } from '@azure/core-tracing';
+import { SpanOptions } from '@azure/core-tracing';
+import { TokenCredential } from '@azure/core-amqp';
+import { TokenType } from '@azure/core-amqp';
+import { WebSocketImpl } from 'rhea-promise';
+
+// @public
+export interface Checkpoint extends PartitionContext {
+    offset: number;
+    sequenceNumber: number;
+}
+
+// @public
+export enum CloseReason {
+    OwnershipLost = "OwnershipLost",
+    Shutdown = "Shutdown"
+}
+
+// @public
+export interface CreateBatchOptions {
+    abortSignal?: AbortSignalLike;
+    maxSizeInBytes?: number;
+    partitionId?: string;
+    partitionKey?: string;
+}
+
+// @public
+export interface EventData {
+    body: any;
+    properties?: {
+        [key: string]: any;
+    };
+}
+
+// @public
+export interface EventDataBatch {
+    batchMessage: Buffer | undefined;
```

Comment:
`batchMessage` is what gets finally "sent". Should this be "internal" as well?

Reply:
+1 on marking it as internal.

Reply:
Done.